spec/requests/api/govpay_webhooks_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Govpay webhooks API" do
+
+  before { allow(WasteCarriersEngine::FeatureToggle).to receive(:active?).with(:api).and_return(true) }
+
+  describe "POST /bo/api/govpay_webhooks/signature" do
+    it "returns a valid hexadecimal value" do
+      params = { foo: :bar }
+
+      post "/bo/api/govpay_webhooks/signature", params: params.to_json, headers: { "CONTENT_TYPE" => "application/json" }
+
+      expect(response.body.match(/^[0-9A-F]+$/i)).to be_present
```

Comment:
This does test only that some hexadecimal response is being returned, but does not test whether that response is correct or not.

Reply:
It could be something as simple as this

params = {param1: 'one', param2: 'two}
post "/bo/api/govpay_webhooks/signature", params: params.to_json, headers: { "CONTENT_TYPE" => "application/json" }
expect(response.body).to eq('345g2dee3qw2e23321')